src/repror/cli/cli.py

```diff
@@ -134,25 +134,27 @@ def rebuild_recipe(
             os.environ["RATTLER_BUILD_BIN"] = str(rattler_build_exe)
         recipes_to_rebuild = build.recipes_for_names(recipe_names)
         rebuild.rebuild_recipe(
-            recipes_to_rebuild, tmp_dir, force_rebuild, patch, actions_url
+            recipes_to_rebuild, Path(tmp_dir), force_rebuild, patch, actions_url
         )
 
 
 @app.command()
-def merge_patches(update_remote: Annotated[Optional[bool], typer.Option()] = False):
+def merge_patches(update_remote: Annotated[Optional[bool], typer.Option()] = None):
```

Comment:
can we use Annotated[bool], ..] = False, and skip later part of:
`if not update_remote` ?

Reply:
No, apparently this default assign does not work for pyright.